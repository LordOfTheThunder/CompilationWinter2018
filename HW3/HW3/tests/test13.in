void TestBoolA() {
	bool a = true;
}
void TestIntA() {
	int a = 33;
}

void main() { print("Max is cool"); }
