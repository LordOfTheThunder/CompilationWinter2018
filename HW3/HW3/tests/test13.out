---end scope---
a BOOL 1
---end scope---
a INT 1
---end scope---
print (STRING)->VOID 1
printi (INT)->VOID 1
TestBoolA ()->VOID 1
TestIntA ()->VOID 1
